spec/resource/application_spec.rb

```diff
@@ -654,6 +654,19 @@ def create_organization_account_store_mapping(organization, account_store)
       end
     end
 
+    context 'exhange id site token for access_token' do
```

Comment:
Looks like this test is only for asserting failures. Can we get tests for verifying the happy paths?